app/models/parrot.rb

```diff
@@ -6,7 +6,7 @@ class Parrot < ActiveRecord::Base
 
   validates :color_id, presence: true
   validates :sex,      presence: true
-  validates :age,      inclusion: { in: 0..240, message: 'Age must be between 0 and 240' }
+  validates :age,      inclusion: { in: 1..240, message: 'Age must be between 1 and 240' }
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.